tests/test_runners/test_maven.py

```diff
@@ -28,6 +28,21 @@ def test_subset(self):
 
     @responses.activate
     @mock.patch.dict(os.environ, {"LAUNCHABLE_TOKEN": CliTestCase.launchable_token})
+    def test_subset_from_file(self):
+        result = self.cli('subset', '--target', '10%', '--session',
+                          self.session, 'maven', "--from-file", str(self.test_files_dir.joinpath("list_1.txt")), "--from-file", str(self.test_files_dir.joinpath("list_2.txt")))
+        self.assertEqual(result.exit_code, 0)
+
+        payload = json.loads(gzip.decompress(
+            responses.calls[0].request.body).decode())
+
+        expected = self.load_json_from_file(
+            self.test_files_dir.joinpath('subset_from_file_result.json'))
+
+        self.assert_json_orderless_equal(expected, payload)
+
+    @ responses.activate
```

Comment:
[must]
remove space between @ and strings.
`@responses.activate`

Reply:
same for others

Reply:
https://github.com/launchableinc/cli/pull/410/commits/2cc3c94ee0bb3057bf8af9d0fabcc58721a6fc4c